pkg/manifests/manifests.go

```diff
@@ -953,6 +953,18 @@ func (f *Factory) NodeExporterServiceMonitor() (*monv1.ServiceMonitor, error) {
 	return sm, nil
 }
 
+func (f *Factory) updateNodeExporterArgs(args []string) []string {
+	if f.config.ClusterMonitoringConfiguration.NodeExporterConfig.Collectors.CpuFreq.Enabled {
+		args = unsetArg(args, "--no-collector.cpufreq")
```

Comment:
this line doesn't seem necessary as neither --no-collector.cpufreq shouldn't be set